web/basics/binding_to_a_native_html_element/my_element.html

```diff
@@ -1,6 +1,6 @@
-<polymer-element name="my-element">
+<dom-module name="my-element">
   <template>
-    <div><input type="range" value="{{val}}"></div>
-    <p>{{val}}</p>
+  	<div><input type="range" value="{{val}}"></div>
```

Comment:
To get `val` to change with the input, you will want to do `{{val::change}}`. See https://github.com/dart-lang/polymer-dart/wiki/data-binding-syntax#two-way-binding-to-native-elements